src/Utilities/PaymentHelpers.res

```diff
@@ -1717,3 +1717,150 @@ let paymentIntentForPaymentSession = (
     (),
   )
 }
+
+let fetchSavedPaymentMethodList = (
+  ~ephemeralKey,
+  ~endpoint,
+  ~optLogger,
+  ~switchToCustomPod,
+  ~isPaymentSession=false,
+) => {
+  open Promise
+  let headers = [("Content-Type", "application/json"), ("api-key", ephemeralKey)]
+  let uri = `${endpoint}/customers/payment_methods`
+  logApi(
+    ~optLogger,
+    ~url=uri,
+    ~apiLogType=Request,
+    ~eventName=CUSTOMER_PAYMENT_METHODS_CALL_INIT,
```

Comment:
```suggestion
    ~eventName=SAVED_PAYMENT_METHODS_CALL_INIT,
```
Lets have separate event name as it otherwise becomes ambiguous with `CUSTOMER_PAYMENT_METHODS_CALL_INIT`